source/includes/administration/_environments.md

```diff
@@ -425,6 +425,16 @@ curl -X POST "https://cloudmc_endpoint/v2/environments/[environment-id]/members"
 }
 ```
 
+```shell
+# Response body example
+```
+```json
+{
+  "taskId": "a82113b9-7f76-4354-ae27-b18b0204d50f",
```

Comment:
with the CRUD operation now being asynchronous, i am curious to know how we will show the added role binding if its no longer returned in the response. It's probably fine with the `system-activity-listener` which can refresh the list on the SUCCESS event of `environment_members.create`